packages/wallet/package.json

```diff
@@ -30,7 +30,7 @@
     "tsdx": "^0.14.1"
   },
   "dependencies": {
-    "@decentralized-identity/ion-sdk": "^0.5.0",
+    "@gjgd/ion-sdk": "0.5.2",
```

Comment:
@gjgd any unless you intend to maintain this, i suggest we just move your fork here... I don't like to be dependent on things we can't update easily here... which I guess includes ion sdk given your comments on the PR.